scripts/skip-upgrade-tests.txt

```diff
@@ -68,5 +68,7 @@ vcd.TestAccVcdVappNetwork_Nat-Update.tf v2.8.0 "nat_enabled and firewall_enabled
 vcd.TestAccVcdVappOrgNetworkDS.tf v2.8.0 "nat_enabled and firewall_enabled were moved to respective resources vcd_vapp_nat_rules and vcd_vapp_firewall_rules"
 vcd.TestAccVcdVappOrgNetwork_Fenced.tf v2.8.0 "nat_enabled and firewall_enabled were moved to respective resources vcd_vapp_nat_rules and vcd_vapp_firewall_rules"
 vcd.TestAccVcdVappOrgNetwork_NotFenced.tf v2.8.0 "nat_enabled and firewall_enabled were moved to respective resources vcd_vapp_nat_rules and vcd_vapp_firewall_rules"
-TestAccVcdVappOrgNetwork_Fenced-Update.tf v2.8.0 "nat_enabled and firewall_enabled were moved to respective resources vcd_vapp_nat_rules and vcd_vapp_firewall_rules"
-TestAccVcdVappOrgNetwork_NotFenced-Update.tf v2.8.0 "nat_enabled and firewall_enabled were moved to respective resources vcd_vapp_nat_rules and vcd_vapp_firewall_rules"
+vcd.TestAccVcdVAppEmptyVm.tf v2.9.0 "connected" default is true, but for "none" networks connect always return false from API.
+vcd.TestAccVcdVAppVmDhcpWait.tf v2.9.0 "connected" default is true, but for "none" networks connect always return false from API.
+vcd.TestAccVcdVAppVmDhcpWait-step1.tf v2.9.0 "connected" default is true, but for "none" networks connect always return false from API.
+vcd.TestAccVcdVAppVmMultiNIC.tf v2.9.0 "connected" default is true, but for "none" networks connect always return false from API.
```

Comment:
Nits:
- All other sentences do **not** end with a dot
- All other sentences are double quoted (not sure if that has any impact or not)

CC: @dataclouder

Reply:
changed